src/SigSpec.CodeGeneration.TypeScript/SigSpecToTypeScriptGenerator.cs

```diff
@@ -27,28 +27,27 @@ public CodeArtifactCollection GenerateArtifacts(SigSpecDocument document)
             {
                 var hubModel = new HubModel(hub.Key, hub.Value, resolver);
                 var template = _settings.TypeScriptGeneratorSettings.TemplateFactory.CreateTemplate("TypeScript", "Hub", hubModel);
-                artifacts.Add(new CodeArtifact(hubModel.Name, CodeArtifactType.Class, CodeArtifactLanguage.TypeScript, template));
+                artifacts.Add(new CodeArtifact(hubModel.Name, CodeArtifactType.Class, CodeArtifactLanguage.TypeScript, CodeArtifactCategory.Undefined, template));
```

Comment:
Do we need a new category?

Reply:
I think Client should be good for this, I fixed this.